lib/rspec/mocks/object_reference.rb

```diff
@@ -42,44 +42,67 @@ def self.anonymous_module?(mod)
     # Used when an object is passed to `object_double`.
     # Represents a reference to that object.
     #
-    # @private
+    # @param object
     class DirectObjectReference
       def initialize(object)
         @object = object
       end
 
+      # The objects desription (`#inspect`).
+      # @return String
       def description
         @object.inspect
       end
 
+      # Not relevant to a direct object reference.
```

Comment:
It might be worth saying, "Defined for interface parity with the other object reference implementations." to explain why we have this at all.